scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -252,7 +269,8 @@ function Initialize-IoTEdge {
         [String] $Username,
 
         # Password used to access the container registry and pull the IoT Edge Agent image.
-        [SecureString] $Password
+        [SecureString] $Password,
+        [switch] $Force=$false
```

Comment:
```suggestion
        # flag to force installation
        [switch] $Force=$false
```

Reply:
Why is this parameter added? Needs to be in the commit message as well as the helptext.

Reply:
the issue here is that iot-core can't uninstall a .cab.  so, to upgrade you need to upgrade the cab and re-run initialize.  initialize should be idempotent anyway imho.  so, that should be ok.  rather then add a -force I can add an iotcore check to skip the error @line 331-336.  would that be better?

Reply:
Why does `Initialize-` need to be re-run?

Reply:
in my case, it was because I was iterating on script changes.  but, in theory if a newer version of the cab has additional or different setup requirements you can't uninstall and re-install on iot core. you need to upgrade the .cab and then rerun init to get the additional settings or whatever other relevant changes there may be.

Reply:
We'll never require running `Initialize-` again. It's impossible for non-Windows because updates happen transparently there and `/etc/iotedge/config.yaml` is never re-generated. So you don't have to worry about that.

Reply:
what about when we switched to UDS and had to add functionality to create the new socket folders?  that's the kind of thing i'm talking about.  currently the script expects you to uninstall and re-install for that sort of case.  but, on iotcore you can't do that. if you can't force a re-init then you have to re-flash the device.

Reply:
That happened before GA.

If that happens again, we can worry about it then.